### PR TITLE
Update user guide 'Securely purchasing and storing Monero'

### LINKED ISSUE #922: Update `Securely purchasing and storing Monero` user guide
*This issue was created on gitlab and then migrated here. Only the original post was migrated, not the comments. Please take a look at the discussions on the original Gitlab issue before commenting here: https://repo.getmonero.org/monero-project/monero-site/-/issues/1078*

---

https://web.getmonero.org/resources/user-guides/securely_purchase.html

It's very outdated (last updated in 2017 and starts with "Step 1: Buy Bitcoin")

### PR DESCRIPTION
**Pull request by erciccione**
Closes #922

### CODE REVIEW

**Comment by erciccione** [comment]
^ two previews because one comes from the new netlify account, which is paid and the other one is the old free account, which will be removed soon

File: _i18n/it/resources/user-guides/securely_purchase.md [hunk 1]
```diff
@@ -1,83 +1,43 @@
 {% include disclaimer.html translated="no" translationOutdated="no" %}
 
-## How to purchase Monero and securely store it.
+## Step 1: Acquire Monero
+There are multiple ways to acquire Monero: You can mine it, you can exchange services or goods for it or you can convert other cryptocurrencies and fiat money to XMR. If you choos the latter, the most convenient way to do it is to use an exchange.
```
Comment (luigi1111) [inline _i18n/it/resources/user-guides/securely_purchase.md:4]: choose

File: _i18n/en/resources/user-guides/securely_purchase.md [hunk 2]
```diff
@@ -1,81 +1,43 @@
 {% include disclaimer.html translated="no" translationOutdated="no" %}
 
-This is a guide to purchase and securely store Monero as of June 2017.
+## Step 1: Acquire Monero
+There are multiple ways to acquire Monero: You can mine it, you can exchange services or goods for it or you can convert other cryptocurrencies and fiat money to XMR. If you choos the latter, the most convenient way to do it is to use an exchange.
```
Comment (luigi1111) [inline _i18n/en/resources/user-guides/securely_purchase.md:4]: "choose"

Also Oxford comma after "for it".

Comment (erciccione) [reply _i18n/en/resources/user-guides/securely_purchase.md]: Fixed. Thanks.

**Pull request merged**